Makefile

```
@@ -12,7 +12,7 @@ clean: ## remove build artifacts
 
 .PHONY: test-unit
 test-unit: ## run unit tests, to change the output format use: GOTESTSUM_FORMAT=(dots|short|standard-quiet|short-verbose|standard-verbose) make test-unit 
-	gotestsum -- $(shell go list ./... | grep -vE '/vendor/|/e2e/')
+	gotestsum $(TESTFLAGS) -- $${TESTDIRS:-$(shell go list ./... | grep -vE '/vendor/|/e2e/')}
```

Should this be

```suggestion
	gotestsum $(TESTFLAGS) -- ${$TESTDIRS:-$(shell go list ./... | grep -vE '/vendor/|/e2e/')}
```

(perhaps not; but the nesting is always confusing me :joy:)

---

I don't think so, the `$$` is escaping a single `$` because this is a `Makefile`.

---

Ah 😊 👍